packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/index.md

```diff
@@ -43,56 +43,39 @@ The SDK represents the sign-in flow using a number of different objects:
-- **Method:** Represents a channel for an authenticator, such as using SMS or voice for an authenticator that uses a phone. An authenticator may have zero or more.
+- **Method:** Represents a channel for an authenticator, such as using SMS or voice for an authenticator that uses a phone. An authenticator may have zero or more methods.
 - **Capability:** A user action associated with a remediation, authenticator, or method, such as requesting a new OTP or a password reset.
-- **Field:** Represents a UI element, either a static item, such as a label, or user input, such as a selection list. It includes properties for state information, such as whether the associated value is required. Properties also store the current value of user input field, such as the string for an OTP or the selected choice. Options, or lists of choices, are represented by a collection of fields. A field may contain a form that contains more fields.
+- **Field:** Represents a UI element, either a static item, such as a label, or user input, such as a selection list. It includes properties for state information, such as whether the associated value is required. Properties also store the current value of user input field, such as the string for an OTP or the selected choice. A lists of choices, or **Options**, are represented by a collection of fields. A field may contain a form that contains more fields.
 - **Form:** Contains the fields that represent the user action for a remediation.
-- **Configuration:** Contains the settings used by the SDK to connect to your Okta org app integration.
-- **Client:** Represents the session during the sign-in flow.
+- **InteractionCodeFlow:** Represents the session during the sign-in flow. In the Android SDK it includes the function for requesting the next step in the flow.
 
 
 ## Objects and the flow
 
-<div class="three-quarter">
-
-!["A diagram that shows the main objects associated with each step in the sign-in flow."](/img/mobile-sdk/mobile-idx-objects-and-flow.png)
-
-</div>
+<StackSnippet snippet="objectsandflow" />
 
-The main objects associated with each step in the flow are:
+## Manage the sign-in flow
 
-| Sign-in step                       | Objects                         |
-| :--------------------------------- | :------------------------------ |
-| Initialize SDK                     | Configuration <br/> Client      |
-| Request initial step               | Client                          |
-| Receive step                       | Response                        |
-| Check completion, cancel, or error | Response <br/> Remediation      |
-| Gather user input                  | Remediation <br/> Authenticator |
-| Send input                         | Remediation                     |
-| Done                               | Response                        |
+An object that manages the sign-in flow includes the following functionality:
```

Comment:
Just a little unclear here. Is the object that you're referring to one that is in the SDK or one that you have to create? If it's prebuilt, what is it called? If it's not, could you revise this to something like "To create an object that manages the sign-in flow, it must include the following functionality;"

Reply:
How's this:

> To create an object that manages the sign-in flow, include the following functionality: